pywb/apps/rewriterapp.py

```diff
@@ -294,9 +277,10 @@ def render_content(self, wb_url, kwargs, environ):
             error = None
             try:
                 error = r.raw.read()
-                r.raw.close()
-            except:
+            except Exception:
                 pass
+            finally:
+                no_except_close(r, raw=True)
```

Comment:
This can just be
```diff
- no_except_close(r, raw=True)
+ no_except_close(r.raw)
```

to ensure raw object is closed

Reply:
changes applied